pages/scripts/main.js

```diff
@@ -67,3 +68,13 @@ document.addEventListener('DOMContentLoaded', function () {
         }
     });
 });
+
+
+const reportBtn = document.querySelector('.main__report');
+const reportFrm = document.querySelector('.report_form');
+
+reportBtn.addEventListener('click', function () {
+    reportFrm.style.display = "flex";
+    this.style.display = "none";
+});
```

Comment:
I suggest wrapping the "Report a problem..." code within a function next time (and future code that may pollute the global namespace). I believe you know about [ES6 modules?](https://medium.com/@manasamancharla11/understanding-javascript-modules-fba7f037fe49)